s6/ipfs/etc/cont-init.d/20-ipfs

```diff
@@ -19,5 +19,9 @@ if [[ ! -d "$IPFS_PATH" ]]; then
   # in IPFS logs
 fi
 
+if [[ ! -z "$IPFS_MIGRATE_FS" ]]; then
```

Comment:
i would compare to "true", not check emptiness. I think that's a better dx, certainly won't fire at MIGRATE_IPFS=false